language-design.md

```diff
@@ -18,6 +18,53 @@ For each quote, describe what it was about the quote that led you pick it.
 **Response**
 
 
+1. "Many brilliant and talented people have contributed to R over the years, many
+of whom are renowned computer scientists for whom I have tremendous respect!
+But a computer scientist is not the same as a software engineer." [Smith, 2016]
+
+We picked this quote because while most of what we have been reading has been 
+from professors and researchers who study the design and implementation of 
+languages, we rarely get a voice from an "outsider" in this respect - someone 
+who just wants to use a language for a purpose and not to analyze its history 
+and design decisions. We think the goal of a well-designed and "good" DSL is to 
+be a tool that accomplishes the domain-specific goals easily or intuitively. 
+Smith does not think R is not well-designed, and traces the cause to the lack 
+of software engineers contributing to R.
```

Comment:
But do they necessarily have to be at odds? A language should be well-designed for its domain. If we call R a DSL, is it well-designed for statistics? If we call R a GPL, is it well-designed for general purposes?

In general, we can call a language well designed if it is easy to use/grow, and the design decisions make sense when considering a target audience. This doesn't necessarily have to vary between industry and academia, however the domain might only encompass one of them.